doc/plugins/jsdoc-foam-plugin.js

```diff
@@ -409,6 +415,15 @@ exports.astNodeVisitor = {
     if ( getDefinitionType(node) ) {
       var defType = getDefinitionType(node);
       var className = getNodePropertyNamed(node, 'name');
+
+      if (typeof className === 'object') {
```

Comment:
Doesn't follow FOAM coding standard. Should be (spaces added):
if ( typeof className === 'object' ) {